external-crates/move/crates/move-cli/tests/build_tests/migration_with_blocking_errors/args.exp

```diff
@@ -0,0 +1,26 @@
+Command `migrate`:
+Package toml does not specify an edition. As of 2024, Move requires all packages to define a language edition.
+
+Please select one of the following editions:
+
+1) 2024.alpha
+2) legacy
+
+Selection (default=1): Recorded edition in 'Move.toml'
+
+Would you like the Move compiler to migrate your code to Move 2024? (Y/n) 
+Generated changes . . .
+BUILDING A
+Unable to generate migration patch due to compilation errors:
+[0m[1m[38;5;9merror[E04007][0m[1m: incompatible types[0m
```

Comment:
More of a nit: any way to have this produce no-color diagnostics in the exp file here?

Reply:
I do not see an obvious way, any suggestions?

Reply:
I will try some things